kafka/1.7/Dockerfile

```diff
@@ -55,6 +55,13 @@ RUN mkdir $KAFKA_HOME/config.orig &&\
 RUN rm -f $KAFKA_HOME/libs/*-{sources,javadoc,scaladoc}.jar* &&\
     rm -r $KAFKA_HOME/site-docs
 
+#
+# The kafka-run-class.sh script generates the classpath for launching Kafka-related JVM, with entries
+# containing the pattern "/bin/../libs", which fails to be resolved properly in some
```

Comment:
Can you adjust the comment so it matches the actual situation here?

Reply:
@gunnarmorling This comment is not just copy/paste but was updated. What's wrong with it?

Reply:
It's just that the regexp could need some explanation, as it's different from the one used for ZK, which literally replaced "/bin/../libs" with "/libs", whereas here you apply some normalization via `realpath`.